common/changes/@uifabric/date-time/textFieldFix_2019-05-29-17-01.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/date-time",
+      "comment": "TextField: adds aria-labelledby to input",
```

Comment:
@staciecarroll, I think these change files are left over from your previous PR. Best to remove them from this contribution.